groupcache.go

```diff
@@ -271,10 +286,7 @@ func (g *Group) load(ctx Context, key string, dest Sink) (value ByteView, destPo
 				return value, nil
 			}
 			g.Stats.PeerErrors.Add(1)
-			// TODO(bradfitz): log the peer's error? keep
-			// log of the past few for /groupcachez?  It's
-			// probably boring (normal task movement), so not
-			// worth logging I imagine.
+			logErr(err)
```

Comment:
Good point!  Just wondering how specific `errLogFn           func(error)` should be ?

```
logErr(fmt.Errorf("g.getFromPeer(ctx, peer, key), ctx: %+v, peer: %+v, key: %+v, err: %+v", ctx, peer, key, err))
```
or 
```
logErr(ctx, peer, key, err)
```